TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/netty/ExpiringSubstitutableItemPool.java

```diff
@@ -319,8 +319,6 @@ private String toStringOnThread() {
             sb.append(", numInProgressItems=").append(inProgressItems.size());
             sb.append(", numReadyItems=").append(readyItems.size());
         }
-        sb.append(", inProgressItems=").append(inProgressItems);
-        sb.append(", readyItems=").append(readyItems);
```

Comment:
This is already printed above if able to. Calling it here can create a concurrency issue if inProgressItems is being modified while being printed.